test/unit/dom/dom.js

```diff
@@ -314,6 +315,57 @@ describe('skatejs-named-slots dom', function () {
       expect(host.outerHTML).to.equal('<div><div></div></div>');
       expect(host.childNodes.length).to.equal(1);
       expect(slot.getAssignedNodes().length).to.equal(1);
+
+      // check normal behaviour of the 'outerHTML' property
+      root.innerHTML = '<div><div></div><div></div></div>';
+      root.childNodes[0].childNodes[0].outerHTML = '<p></p>';
+      expect(root.innerHTML).to.equal( '<div><p></p><div></div></div>');
+
+      root.innerHTML = '<div><div></div><div></div></div>';
+      root.childNodes[0].outerHTML = '<p></p>';
+      expect(root.innerHTML).to.equal('<p></p>');
+
+      host.innerHTML = '<div><div></div><div></div></div>';
+      host.childNodes[0].childNodes[0].outerHTML = '<p></p>';
+      expect(host.innerHTML).to.equal('<div><p></p><div></div></div>');
+
+      host.innerHTML = '<div><div></div><div></div></div>';
+      host.childNodes[0].outerHTML = '<p></p>';
+      expect(host.innerHTML).to.equal('<p></p>');
+
+      // host has no parentNode so we expect is to throw an error in some browsers, otherwise do nothing
+      const errorMsg = 'Failed to set the \'outerHTML\' property on \'Element\': This element has no parent node.';
+      const errorMsgOpera = 'Failed to call host setter';
+      host.innerHTML = '';
+      expect(host.outerHTML).to.equal('<div></div>');
+
+      if (canPatchNativeAccessors) {
+        try {
+          host.__outerHTML = '';
+        } catch(e) {
+          expect(function() {
+            host.outerHTML = '<p></p>';
+          }).to.throw(Error);
+
+          try {
+            host.outerHTML = '<p></p>';
+          } catch(e) {
+            expect([errorMsg, errorMsgOpera].indexOf(e.message)).to.be.above(-1);
```

Comment:
I imagine this `expect` and the one above it are testing the same thing (it throws an error, and it throws an error with the correct message). It could be simplified a bit if we only had the second assertion – you might need to have some way of requiring a certain number of `expects` to execute though